Image Classifier Unit Test Failure

Image Classifier `text_export_coreml` unit test fails for model `VisionFeaturePrintSceneTest`.

```
=================================== FAILURES ===================================
________________ VisionFeaturePrintSceneTest.test_export_coreml ________________

self = <turicreate.test.test_image_classifier.VisionFeaturePrintSceneTest testMethod=test_export_coreml>

    def test_export_coreml(self):
        filename = tempfile.mkstemp('bingo.mlmodel')[1]
        self.model.export_coreml(filename)
    
        coreml_model = coremltools.models.MLModel(filename)
        self.assertDictEqual({
            'com.github.apple.turicreate.version': tc.__version__,
            'com.github.apple.os.platform': platform.platform(),
            'type': 'ImageClassifier',
            'version': '1'
            }, dict(coreml_model.user_defined_metadata)
        )
        expected_result = 'Image classifier (%s) created by Turi Create (version %s)' % (
                self.model.model, tc.__version__)
>       self.assertEquals(expected_result, coreml_model.short_description)
E       AssertionError: 'Image classifier (VisionFeaturePrint_Scene) created by Turi Create (version 6.0rc0)' != u'Image classifier (visionfeatureprint_scene) created by Turi Create (version 6.0rc0)'

test_image_classifier.py:185: AssertionError
```

--- a/src/python/turicreate/test/test_image_classifier.py
+++ b/src/python/turicreate/test/test_image_classifier.py
@@ -24,7 +24,6 @@ from . import util as test_util
 
 import coremltools
 import numpy as np
-import pytest
 import turicreate as tc
 
 
@@ -207,11 +206,6 @@ class ImageClassifierTest(unittest.TestCase):
             predictions = model.classify("more junk")
 
     def test_export_coreml(self):
-        if self.model.model == "VisionFeaturePrint_Scene":
-            pytest.xfail(
-                "Expected failure until "
-                + "https://github.com/apple/turicreate/issues/2744 is fixed"
-            )
         filename = tempfile.NamedTemporaryFile(suffix=".mlmodel").name
         self.model.export_coreml(filename)
 
@@ -228,7 +222,7 @@ class ImageClassifierTest(unittest.TestCase):
         )
         expected_result = (
             "Image classifier (%s) created by Turi Create (version %s)"
-            % (self.model.model, tc.__version__)
+            % (self.model.model.lower(), tc.__version__)
         )
         self.assertEquals(expected_result, coreml_model.short_description)
 

